src/app/shared/components/loading-overlay/loading-overlay.component.less

```
@@ -0,0 +1,140 @@
+  justify-content: center;
+  animation: pulse-logo 1.4s ease-in-out infinite;
+}
+
+.logo-img {
+  width: 48px;
+  height: auto;
+}
+
+// ── Brand name ─────────────────────────────────────────────────────────────────
+.brand-name {
+  font-family: 'Inter', 'Segoe UI', sans-serif;
```

Stylelint (17.3.0)

[error] Unexpected quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)